bom/pom.xml

```diff
@@ -32,6 +32,9 @@
         <cassandraVersion>2.2.0-rc3</cassandraVersion>
         <!-- Update dependency versions accordingly when updating lettuce  -->
         <lettuceVersion>4.1.Final</lettuceVersion>
+        <rxjavaVersion>1.1.0</rxjavaVersion>
+        <hdrHistogramVersion>2.1.8</hdrHistogramVersion>
```

Comment:
I think the only one needed is rxjava, I'll try removing the other ones

On Tue, Mar 29, 2016 at 12:25 PM, Sanne Grinovero <notifications@github.com>
wrote:

> In bom/pom.xml
> <https://github.com/hibernate/hibernate-ogm/pull/645#discussion_r57707765>
> :
>
> > @@ -32,6 +32,9 @@
> >          <cassandraVersion>2.2.0-rc3</cassandraVersion>
> >          <!-- Update dependency versions accordingly when updating lettuce  -->
> >          <lettuceVersion>4.1.Final</lettuceVersion>
> > +        <rxjavaVersion>1.1.0</rxjavaVersion>
> > +        <hdrHistogramVersion>2.1.8</hdrHistogramVersion>
>
> these are usually testing libraries. Are you sure they are needed?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/hibernate/hibernate-ogm/pull/645/files/21d6fcede5d2f176be6a005a67388e5d2915f592..3f7fd73c5ad0efaa7b5bb9904eae3f9a2cf02bec#r57707765>
>